### PR TITLE
This is my first pull request, Hello Microverse!

### PR DESCRIPTION
**Pull request by RafaelEchart**
This pull request contains all the files for the first draft of my Microverse page and the tests to complete the challenge.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (SajjadAhmad14) [inline .github/workflows/linters.yml:1]: Please add all the files from this code linter [config](https://github.com/microverseinc/linters-config/tree/master/html-css).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```
Comment (SajjadAhmad14) [inline README.md:3]: A professional README file should have steps to run the project locally.

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
-# .github/
\ No newline at end of file
+# .github/
```
Comment (SajjadAhmad14) [inline .gitignore:1]: Please add `node_modules` to gitignore.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Rafael Echart Saltos, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: .github/workflows/linters.yml (hunk 1 above)
Comment (RafaelEchart) [reply .github/workflows/linters.yml:1]: What code should I add? I copied and pasted the code from: https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @RafaelEchart , 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**